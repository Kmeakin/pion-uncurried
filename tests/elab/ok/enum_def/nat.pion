enum Nat {
    Zero,
    Succ(n: Nat),
}

let z = Zero;
let s = Succ;