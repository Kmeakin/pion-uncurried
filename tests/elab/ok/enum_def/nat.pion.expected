enum Nat: Type {
    Zero: Nat,
    Succ(n: Nat): Nat,
}

let z: Nat = Zero;

let s: fn(n: Nat) -> Nat = Succ;
